dcos/package.py

```diff
@@ -1227,6 +1227,13 @@ def options(self, revision, user_options):
 
         config_schema = self.config_json(revision)
         default_options = _extract_default_values(config_schema)
+        if default_options is None:
+            maintainer = self.package_json(revision)['maintainer']
+            msg = ("Error with the package's config.json: {}\n"
```

Comment:
Print the error type (missing required...) before the config.json.

Ideally, you would tell them exactly where 'properties' is missing.  But you should at least say "An object in config.json is missing the required 'properties feature".  It sounds like the property is missing in the outer object as it is.